examples/dataflow-cloud-run/dataflowApp/src/main/java/com/demo/dataflow/model/GoBike.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.demo.dataflow.model;
+
+
+import org.apache.beam.sdk.coders.AvroCoder;
+import org.apache.beam.sdk.coders.DefaultCoder;
+
+import java.util.Map;
+
+@DefaultCoder(AvroCoder.class)
+public class GoBike {
+    private Integer durationSec;
+    private String startTime;
+    private String endTime;
+
+    private Integer startStationId;
+    private String startStationName;
+    private String startStationLatitude;
+    private String startStationLongitude;
+
+    private Integer endStationId;
+    private String endStationName;
+    private String endStationLatitude;
+    private String endStationLongitude;
+
+    private Integer bikerId;
+
+    private String userType;
+    private Integer memeberBirthYear;
+
+    private String memberGender;
+
+    private Boolean bikeShareForAllTrip;
+
+
+    public static GoBike createFromMap(Map<String,String> data) {
+        String[] headers = getHeader();
+        return new GoBike(Integer.parseInt(data.get(headers[0])), data.get(headers[1]), data.get(headers[2]), Integer.parseInt(data.get(headers[3])), data.get(headers[4])
+                ,data.get(headers[5]), data.get(headers[6]), Integer.parseInt(data.get(headers[7])),data.get(headers[8]),
+                data.get(headers[9]), data.get(headers[10]), Integer.parseInt(data.get(headers[11])), data.get(headers[12]), Integer.parseInt(data.get(headers[13])), data.get(headers[14]), Boolean.parseBoolean(data.get(headers[15])));
+    }
+
+    public GoBike() {
+    }
+
+    public GoBike(Integer durationSec, String startTime, String endTime, Integer startStationId, String startStationName, String startStationLatitude, String startStationLongitude, Integer endStationId, String endStationName, String endStationLatitude, String endStationLongitude, Integer bikerId, String userType, Integer memeberBirthYear, String memberGender, Boolean bikeShareForAllTrip) {
+        this.durationSec = durationSec;
```

Comment:
too many params, makes it prone for accidental errors and reduces readability. Possibly replace with builder class and make this constructor private.